src/main/java/hu/dpc/phee/operator/importer/VariableParser.java

```diff
@@ -65,7 +65,6 @@ public VariableParser() {
         transactionRequestParsers.put("quoteId", pair -> pair.getFirst().setPayeeQuoteCode(strip(pair.getSecond())));
         transactionRequestParsers.put("transactionState", pair -> parseTransActionState(pair.getFirst(), pair.getSecond()));
         transactionRequestParsers.put("mpesaChannelRequest", pair -> parseTransactionMpesaRequest(pair.getFirst(), pair.getSecond()));
-        transactionRequestParsers.put("originDate", pair -> pair.getFirst().setStartedAt(new Date(Long.parseLong(pair.getSecond()))));
```

Comment:
@danishjamal104 Why is this removed?

Reply:
because now the start time is processed by the `PROCESS_INSTANCE` value type. And is set at [this](https://github.com/openMF/ph-ee-importer-rdbms/blob/ba0210ee3e57734c65cd16d335c22f13f44ae92d/src/main/java/hu/dpc/phee/operator/importer/InflightTransactionRequestManager.java#L27) place.